api/turing/cluster/servicebuilder/fluentd.go

```diff
@@ -125,17 +125,8 @@ func buildFluentdVolumes(
 		MountPath: secretMountPath,
 	})
 
-	volumes = append(volumes, corev1.Volume{
-		Name: ComponentTypes.CacheVolume,
-		VolumeSource: corev1.VolumeSource{
-			PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
-				ClaimName: cacheVolumePVCName,
-			},
-		},
-	})
```

Comment:
This is removed as the volumes get created automatically by the stateful set resource.

Reply:
how does the stateful set "reuse" a previous volume then?

Reply:
nvm, on second thought, the volume are always deleted with statefulset

Reply:
Ah do you mean the PVCs? That's not done automatically when the stateful set is deleted, which is why there's this other function `DeleteStatefulSetPersistentVolumeClaims` that you commented on below.